map/create-and-save-map/src/main/java/com/esri/samples/create_and_save_map/CreateAndSaveMapController.java

```diff
@@ -213,9 +215,10 @@ private void saveMap() {
 
   /**
    * Display an alert to the user with the specified information.
-   * @param title alert title
+   *
+   * @param title       alert title
```

Comment:
I don't think this much space is needed between the parameter name and description

Reply:
Thanks, I think that was Intellij automatically formatting things. That method is removed now, so no longer applicable.